### ISSUE TITLE
why do you set nModules to 2?

### ISSUE THREAD

**Issue by liuyangwen** [xingyizhou/pytorch-pose-hg-3d#39]
I wonder why you set nModules(residual modules at each hourglass) to 2 rather than 1? Is there any special effect?

**Comment by xingyizhou** [comment]
Hi, 
  Sorry for the late reply. I found it achieve a good training speed - accuracy in the initial hyper-parameter search. It should have a close performance with 8 stacks 1 module.
